# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/libsafe-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- libsafe-2.0-16/src/intercept.c.orig	2005-01-22 12:53:10.000000000 -0800
+++ libsafe-2.0-16/src/intercept.c	2005-01-22 12:47:15.000000000 -0800
@@ -78,8 +78,11 @@
 typedef char *(*getwd_t) (char *buf);
 typedef char *(*gets_t) (char *s);
 typedef char *(*realpath_t) (char *path, char resolved_path[]);
-typedef int (*_IO_vfscanf_t) (_IO_FILE *s, const char *format, _IO_va_list
-	argptr, int *errp);
+typedef int (*_IO_vfscanf_t)
+ (FILE *s,
+   const char *format,
+   va_list argptr,
+   int *errp);
 
 
 
@@ -1443,7 +1446,7 @@
 }
 
 
-int _IO_vfscanf (_IO_FILE *s, const char *format, _IO_va_list argptr, int *errp)
+int _IO_vfscanf (FILE *s, const char *format, va_list argptr, int *errp)
 {
     static _IO_vfscanf_t real_IO_vfscanf = NULL;
     int res, save_count;
